LearningMissionSimulation/AttributeGenerator.cs

```diff
@@ -298,6 +298,65 @@ public static string GetCoverLetter()
             //not implemented yet!
             return "";
         }
+        public static string GetUnitName()
+        {
+            string unitName = GetFirstName();
```

Comment:
The requirement was to use a prefix that contains the unit's type. Your GetUnitName method should take an input parameter: UnitType unitType. You can use a switch statement then to handle all available unit types. My requirement was that the generated random name should start with the prefix defined by the unit type and then followed by 4-digit random code. For example:
1. input parameter UnitType.Module, output value should look like "Module3982"
2. input parameter UnitType.Subject, output value should look like "Subject7629"
You can't use GetFirstName method here.